examples/java-spring-app.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: devx.ibm.com/v1alpha1
 kind: StarterKit
 metadata:
-  name: java-spring app
+  name: java-spring-app
 spec:
   templateRepo:
     templateOwner: IBM
     templateRepoName: java-spring-app
     name: java-spring-app
-    owner: <OWNER>
+    owner: gee4vee
```

Comment:
Can you revert this? You can keep the name updated.

Reply:
Fixed